modules/images-configuration-shortname.adoc

```diff
@@ -0,0 +1,94 @@
+
+If you list public registries under the `containerRuntimeSearchRegistries` parameter, you expose your credentials to all the registries on the list and you risk network and registry attacks. You should always use fully-qualified image names with public registries.
+====
+
+The Machine Config Operator (MCO) watches the `image.config.openshift.io/cluster` resource for any changes to the registries. When the MCO detects a change, it drains the nodes, applies the change, and uncordons the nodes. After the nodes return to the `Ready` state, if the `containerRuntimeSearchRegistries` parameter is added, the MCO and creates a file in the `/etc/containers/registries.conf.d` directory on each node with the listed registries. The file overrides the default list of unqualified search registries in the `/host/etc/containers/registries.conf` file. There is no way to fall back to the default list of unqualified search registries.
+
+The `containerRuntimeSearchRegistries` parameter works only with the Podman and CRI-O container engines and the registries in the list can be used only in pod specs, not in builds and image streams.
+
+.Procedure
+
+. Edit the `image.config.openshift.io/cluster` custom resource:
++
+[source,terminal]
+----
+$ oc edit image.config.openshift.io/cluster
+----
++
+The following is an example `image.config.openshift.io/cluster` CR:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Image
+metadata:
+  annotations:
+    release.openshift.io/create-only: "true"
+  creationTimestamp: "2019-05-17T13:44:26Z"
+  generation: 1
+  name: cluster
+  resourceVersion: "8302"
+  selfLink: /apis/config.openshift.io/v1/images/cluster
+  uid: e34555da-78a9-11e9-b92b-06d6c7da38dc
+spec:
+  allowedRegistriesForImport:
+    - domainName: quay.io
+      insecure: false
+  additionalTrustedCA:
+    name: myconfigmap
+  registrySources:
+    containerRuntimeSearchRegistries: <1>
+    - "reg1.io"
+    - "reg2.io"
+    - "reg3.io"
+    allowedRegistries: <2>
+    - example.com
+    - quay.io
+    - registry.redhat.io
+    - "reg1.io"
+    - "reg2.io"
+    - "reg3.io"
+...
+status:
+  internalRegistryHostname: image-registry.openshift-image-registry.svc:5000
+----
+<1> Specify registries to use with image short names. You should use image short names with only internal or private registries to reduce possible security risks.
+<2> Ensure that any registries listed under `containerRuntimeSearchRegistries` are included in the `allowedRegistries` list.
++
+[NOTE]
+====
+When the `allowedRegistries` parameter is defined, all registries including the `registry.redhat.io` and `quay.io` registries are blocked unless explicitly listed. If you use this parameter, to prevent pod failure, add `registry.redhat.io` and `quay.io` to the `allowedRegistries` list, as they are required by payload images within your environment. 
+====
+
+To check that the registries have been added, use the following command on a node:
++
```

Comment:
The `+` markers are currently rendering (line 83 and 88). I'd suggest making this verification step a numbered step, which would fix the rendering issue.
> . To check that the registries have been added, use the following command on a node: